qa/tasks/ceph_manager.py

```diff
@@ -637,7 +638,6 @@ def grow_pool(self):
         Increase the size of the pool
         """
         pool = self.ceph_manager.get_pool()
-        orig_pg_num = self.ceph_manager.get_pool_pg_num(pool)
```

Comment:
why are you removing this line?

Reply:
```
➜  ceph git:(wip-ec-below-min-size) ✗ grep -nri "orig_pg_num" qa/tasks/ceph_manager.py
➜  ceph git:(wip-ec-below-min-size) ✗ 
```
it seems `orig_pg_num` is a variable not used.

Reply:
ahh, i see. i forgot to remove it in c0f0cde399a80cea617c115417e5390a63ec674e .

thanks!